geoportal/c2cgeoportal_geoportal/scripts/theme2fts.py

```diff
@@ -153,25 +185,60 @@ def __init__(self, session: Session, settings: pyramid.config.Configurator, opti
                 self._[lang] = gettext.NullTranslations()
                 print(f"Warning: {e} (language: {lang})")
 
+        if options.debug:
+            print("Import before query")
+
         query = self.session.query(Interface)
+
+        if options.debug:
+            print("Import got query")
```

Comment:
```suggestion
            print("Import got query interface")
```
Note that we just built an SQLAlchemy object!